build-aux/snap/snapcraft.yaml

```diff
@@ -127,3 +128,26 @@ parts:
       cp -a fc-cache-bionic $SNAPCRAFT_PART_INSTALL/bin/fc-cache-v7
     prime:
       - bin/fc-cache-v7
+  apparmor:
+    plugin: autotools
+    build-packages: [ bison, flex, gettext ]
+    source: https://launchpad.net/apparmor/3.0/3.0.3/+download/apparmor-3.0.3.tar.gz
+    override-build: |
+      cd libraries/libapparmor
+      ./autogen.sh
+      ./configure --prefix=/usr
```

Comment:
probably also worth adding --disable-man-pages --disable-perl --disable-python --disable-ruby

Reply:
Thanks - done in https://github.com/snapcore/snapd/pull/9939/commits/35acd6d3265a3aa7489c1e8e45735038807073bb